.github/workflows/build.yml

```
@@ -25,18 +25,23 @@ jobs:
       - uses: actions/checkout@v4.1.1
       - name: Build debug
         run: cargo build
-      - name: Run system tests
+      - name: Prepare system tests
         run: |
+          echo "Retrieved github.event_name = ${{ github.event_name }}"
           mkdir -p target/robot_tests_result
           chown vscode target/robot_tests_result
-          if [ ${{ github.event_name == 'pull_request' }} ]; then
-            SYSTEM_TEST_EXCLUDE_TAG="non_execution_during_pull_request_verification"
-            echo "Run is a PR verification, system tests tagged with '$SYSTEM_TEST_EXCLUDE_TAG' are NOT executed and skipped."
-            su vscode -c "tools/run_robot_tests.sh --exclude $SYSTEM_TEST_EXCLUDE_TAG tests"
-          else
+      - name: Run system tests in pull request verification
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
+          SYSTEM_TEST_EXCLUDE_TAG="non_execution_during_pull_request_verification"
+          echo "Run is a PR verification, system tests tagged with '$SYSTEM_TEST_EXCLUDE_TAG' are NOT executed and skipped."
+          su vscode -c "tools/run_robot_tests.sh --exclude $SYSTEM_TEST_EXCLUDE_TAG tests"
+      - name: Run all system tests
+        if: ${{ github.event_name != 'pull_request' }}
+        run: |
+            echo "Retrieved github.event_name = ${{ github.event_name }}"
             echo "Run is not a PR verification, all system tests are executed."
```

Do we need the first line here? Maybe we can combine the two:
```suggestion
            echo "Run is ${{ github.event_name }}, all system tests are executed."
```